docker-compose.yml

```diff
@@ -18,12 +18,13 @@
 #
 # For those usages not covered by the GNU Affero General Public License
 # please contact with sc_support at telefonica dot com
-
```

Comment:
In which way the new functionality about env vars is related with the changes in this docker-compose.yml file?